src/grid-gutter.json

```
@@ -0,0 +1,29 @@
+{
+  "grid-gutter": {
+    "xs": {
+      "value": "8px",
```

We should not duplicate other sizing tokens when creating new tokens. It is recommended to update using an alias to the source of truth. 

```suggestion
      "value": "{size.100.value}",
```